app/assets/javascripts/discourse/app/components/user-menu/menu.js

```diff
@@ -1,40 +1,105 @@
 import GlimmerComponent from "discourse/components/glimmer";
 import { tracked } from "@glimmer/tracking";
 import { action } from "@ember/object";
+import UserMenuTab from "discourse/lib/user-menu/tab";
 
 const DEFAULT_TAB_ID = "all-notifications";
 const DEFAULT_PANEL_COMPONENT = "user-menu/notifications-list";
 
+const CORE_TOP_TABS = [
+  class extends UserMenuTab {
+    get id() {
+      return DEFAULT_TAB_ID;
+    }
+
+    get icon() {
+      return "bell";
+    }
+
+    get panelComponent() {
+      return DEFAULT_PANEL_COMPONENT;
+    }
+  },
+
+  class extends UserMenuTab {
+    get id() {
+      return "replies";
+    }
+
+    get icon() {
+      return "reply";
+    }
+
+    get panelComponent() {
+      return "user-menu/replies-notifications-list";
+    }
+  },
+
+  class extends UserMenuTab {
+    get id() {
+      return "mentions";
+    }
+
+    get icon() {
+      return "at";
+    }
+
+    get panelComponent() {
+      return "user-menu/mentions-notifications-list";
+    }
+  },
+
+  class extends UserMenuTab {
+    get id() {
+      return "likes";
+    }
+
+    get icon() {
+      return "heart";
+    }
+
+    get panelComponent() {
+      return "user-menu/likes-notifications-list";
+    }
+
+    get shouldDisplay() {
+      return !this.currentUser.no_likes_notifications;
```

Comment:
Not sure about naming here `no_likes_notifications` reads more like "I didn't get any like notifications" than "I disabled likes notifications"

Maybe "likes_notifications_disabled" would be more explicit?

Reply:
done!